internal/server/landing_page.go

```diff
@@ -248,7 +248,10 @@ func fetchBtData(
 	// Populate result from landing page cache
 	result := map[string]map[string]*ObsTimeSeries{}
 	for dcid, data := range dataMap {
-		landingPageData := data.(*LandingPageData)
+		landingPageData, ok := data.(*LandingPageData)
```

Comment:
To confirm, are these changes necessarily related to bigqueryOnly mode?

If so, why don't we return an ERROR in bigTableReadRowsParallel interface? Wouldn't that be simpler (since we already handle it)?

Reply:
This is not entirely related to that. In general, we want to check if the conversion is ok to prevent crash, and we did in some places (to check the nil pointer).